source/_docs/platform-resources.md

```diff
@@ -1,75 +1,275 @@
 ---
-title: Platform Resources by Plan
+title: Platform Resources and Features by Plan
+plancomparison: true
 description: Get detailed information about platform resources for your Drupal or WordPress site.
 keywords: site, RAM, pantheon, backup, plane
 categories: [developing]
 tags: [code, debug, platform]
 ---
-The platform resources provided to your website depend on your current plan. Pantheon can scale instantly, so changing your service level will immediately change your resources to the values for the new plan, as shown in the table below.
+The platform resources and features provided to your website depend on your current plan. Pantheon can scale instantly, so changing your service level will immediately change your resources to the values for the new plan, as shown in the table below.
 
-## Platform Resources
+## Resources and Features by Plan
 
-<table class="table table-condensed table-bordered">
-    <thead class="thead-inverse">
+<table class="table  table-bordered table-responsive table-striped plan-comparison">
+    <thead>
       <tr>
-        <th scope="row" class="thead-inverse"></th>
-        <th>Personal</th>
-        <th>Professional</th>
-        <th>Business</th>
-        <th>Elite</th>
+        <th class="subscription"></th>
+        <th class="subscription">Personal</th>
```

Comment:
This column seems a bit squished and some of the content wraps to the second line